docs(aws): add AWS console UI steps

aleon1220 opened the pull request:
1-A proposal to have a section option 3 that describes the steps to be taken in AWS console manually. I am happy to add the screenshots that match the generic names 

2-A new Halyard section with description of the general steps that should be taken inside halyard

aleon1220 committed:
Draft proposal option 3 and Halyard step structure

1-A proposal to have a section option 3 that describes the steps to be taken in AWS console manually. I am happy to add the screenshots that match the generic names 

2-A new Halyard section with description of the general steps that should be taken inside halyard

317brian commented on setup/install/providers/aws/aws-ec2.md:
```diff
@@ -79,8 +79,24 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
+Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations.
+
+## Halyard Configurations
+After the AWS IAM user, roles, policies and trust relationship has been set up, the next step is to add the AWS configurations to spinnaker via Halyard CLI.
```
capital 'S' for Spinnaker

317brian commented on setup/install/providers/aws/aws-ec2.md:
```diff
@@ -79,8 +79,24 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
+After the AWS IAM user, roles, policies and trust relationship has been set up, the next step is to add the AWS configurations to spinnaker via Halyard CLI.
+
+The steps are as following:
+* Access the Halyard Pod
```
These steps should be an ordered list. Also, can we be explicit about each step, e.g. 

```
Apply the configurations to spinnaker
```

should become

```
4. Apply the configurations to Spinnaker: `hal deploy apply`. 
```

Period at the end of each list item too.

aleon1220 reviewed (commented):
thank you. I agree with the changes

aleon1220 committed:
Merge branch 'master' into patch-1

aleon1220 reviewed (commented):
It looks good to me.

aleon1220 committed:
Merge branch 'master' into patch-1

317brian commented:
Hi @aleon1220, there isn't a commit with my suggested updates.

dorbin committed:
Merge branch 'master' into patch-1

aleon1220 committed:
Updated based on suggested changes

Changes have been suggested by user @brian-armory 

I am sorry i would perform better if i know a bit more about the markdown editing with Github interface.
perhaps i should use a suggested plugin for VSCode.

please let me know

aleon1220 committed:
Merge branch 'master' into patch-1

317brian committed:
Update aws-ec2.md

317brian committed:
Merge branch 'master' into patch-1
